iina/MainWindowController.swift

```diff
@@ -1895,7 +1895,7 @@ class MainWindowController: PlayerWindowController {
     NSLayoutConstraint.activate(constraintsH)
     NSLayoutConstraint.activate(constraintsV)
     var viewController = viewController
-    viewController.downShift = titleBarView.frame.height
+    viewController.downShift = TitleBarHeightNormal - 14
```

Comment:
I've been trying to remove magic numbers from MainWindowController. Can the 14 be replaced with a reference to a component's property?

Reply:
Tried... See comment below.